### PR TITLE
All the changes of routes has been updated #39

### LINKED ISSUE #39: [FEATURE] Create router
Description
Currently, both components are showing in the app when loading, the `<Content />` and the `<Hero />`.

Create a router, that, when the homepage shows the `<Hero />` component (as well as when clicking on Home in the navigation bar), and when clicking on Courses, the `<Content />` component is showing.

```[tasklist]
### Tasks
- [ ] Implement routing for Home and Courses
```

**Note:** Decide if you are going to use Next routing system or the new react-router. Explain why in the PR.

### PR DESCRIPTION
**Pull request by Ahsankhalid618**
[ issue closes #39 ]
I've made some changes to the content and hero components. Plus, I've cleaned up the way we do routing in our Next.js app. The coolest part is now you can easily hop between the home/hero section by just hitting the "Home" button in the navbar, and when you're ready to dive into the content section, just hit the "Courses" button – it's way more user-friendly. Next.js makes routing a breeze, even for nested pages, so it's a win-win. All of these tweaks are aimed at making our web app more functional and user-friendly, making the whole experience a lot more engaging and user-focused.

image

### CODE REVIEW

File: pages/Header.js [hunk 1]
```diff
@@ -1,15 +1,17 @@
...
+import Link from 'next/link';
 
 const Header = () => {
   return (
     <header className='nav-header'>
         <nav>
             <ul>
                 <li>
-                    <a href='#'>Home</a>
+
```
Comment (YuriDevAT) [inline pages/Header.js:10]: Remove unnecessary empty line.

File: pages/Header.js [hunk 2]
```diff
@@ -1,15 +1,17 @@
 import React from 'react';
+import Link from 'next/link';
 
 const Header = () => {
   return (
     <header className='nav-header'>
         <nav>
```
Comment (YuriDevAT) [inline pages/Header.js:7]: For consistency reasons stick to spaces: 2 as the rest of the files.

File: pages/Header.js [hunk 3]
```diff
@@ -1,15 +1,17 @@
...
             <ul>
                 <li>
-                    <a href='#'>Home</a>
+                    
+                    <Link href="/">Home</Link>
```
Comment (YuriDevAT) [inline pages/Header.js:11]: Nice implementation of NextJS Linking and Navigation 👏🏽

File: pages/index.js [hunk 4]
```diff
@@ -1,9 +1,9 @@
 import Head from "next/head";
-import Content from "../components/content";
-import Header from "./components/Header";
-import Hero from "../components/hero/Hero";
+import Content from "./courses";
+import Header from "./Header";
+import Hero from "./home/Hero";
 
-export default function Home() {
+export default function Myapp() {
```
Comment (YuriDevAT) [inline pages/index.js:6]: What's the reason of changing the name of the home component? Also, components are always PascalCase, as this is suggested by React docs itself.

**Review by YuriDevAT — CHANGES_REQUESTED** [review]
Thanks for your PR and contributing to the projects 👍🏽 I left suggestions which should be implemented before the PR can be merged.

**Comment by YuriDevAT** [comment]
Can you share some pictures of your local output? On my side, on the home route, both components are still shown, on the courses route, the correct component is shown but the header disappears.

### Home View
Screenshot 2023-10-20 at 17-38-40 Create Next App

### Courses View
Screenshot 2023-10-20 at 17-38-55 Screenshot

File: pages/Header.js (hunk 1 above)
Comment (Ahsankhalid618) [reply pages/Header.js:10]: Solved. Kindly review again

File: pages/Header.js (hunk 2 above)
Comment (Ahsankhalid618) [reply pages/Header.js:7]: Solved. Kindly review again

File: pages/index.js (hunk 4 above)
Comment (Ahsankhalid618) [reply pages/index.js:6]: Solved. I have changed the name properly. Kindly review again

**Comment by Ahsankhalid618** [comment]
I have solved all the issues. Kindly review it again for successful merge and if you merge this then tag hacktoberfest-accepted tag.

**Comment by YuriDevAT** [comment]
Kindly read the [contribution guideline](https://github.com/YurisCodingClub/accessibility-mentor/blob/main/CONTRIBUTING.md).

**Pull request closed without merging**